Index: glib-2.26.1/gthread/gthread-posix.c
===================================================================
--- glib-2.26.1.orig/gthread/gthread-posix.c	2011-08-04 12:09:18.729931454 -0700
+++ glib-2.26.1/gthread/gthread-posix.c	2011-08-04 12:12:21.199931544 -0700
@@ -345,7 +345,8 @@
     struct sched_param sched;
     posix_check_cmd (pthread_attr_getschedparam (&attr, &sched));
     sched.sched_priority = g_thread_priority_map [priority];
-    posix_check_cmd_prio (pthread_attr_setschedparam (&attr, &sched));
+    /*posix_check_cmd_prio (pthread_attr_setschedparam (&attr, &sched));*/
+    posix_check_cmd_prio(pthread_attr_setschedpolicy(&attr, SCHED_OTHER));
   }
 # else /* G_THREADS_IMPL_DCE */
   posix_check_cmd_prio
