Index: gettext-0.18.1.1/gettext-runtime/configure
===================================================================
--- gettext-0.18.1.1.orig/gettext-runtime/configure
+++ gettext-0.18.1.1/gettext-runtime/configure
@@ -11962,8 +11962,8 @@ linux* | k*bsd*-gnu)
   version_type=linux
   need_lib_prefix=no
   need_version=no
-  library_names_spec='${libname}${release}${shared_ext}$versuffix ${libname}${release}${shared_ext}$major $libname${shared_ext}'
-  soname_spec='${libname}${release}${shared_ext}$major'
+  library_names_spec='${libname}${release}${shared_ext}'
+  soname_spec='${libname}${release}${shared_ext}'
   finish_cmds='PATH="\$PATH:/sbin" ldconfig -n $libdir'
   shlibpath_var=LD_LIBRARY_PATH
   shlibpath_overrides_runpath=no
Index: gettext-0.18.1.1/gettext-runtime/libasprintf/configure
===================================================================
--- gettext-0.18.1.1.orig/gettext-runtime/libasprintf/configure
+++ gettext-0.18.1.1/gettext-runtime/libasprintf/configure
@@ -11219,8 +11219,8 @@ linux* | k*bsd*-gnu)
   version_type=linux
   need_lib_prefix=no
   need_version=no
-  library_names_spec='${libname}${release}${shared_ext}$versuffix ${libname}${release}${shared_ext}$major $libname${shared_ext}'
-  soname_spec='${libname}${release}${shared_ext}$major'
+  library_names_spec='${libname}${release}${shared_ext}'
+  soname_spec='${libname}${release}${shared_ext}'
   finish_cmds='PATH="\$PATH:/sbin" ldconfig -n $libdir'
   shlibpath_var=LD_LIBRARY_PATH
   shlibpath_overrides_runpath=no
@@ -14679,8 +14679,8 @@ linux* | k*bsd*-gnu)
   version_type=linux
   need_lib_prefix=no
   need_version=no
-  library_names_spec='${libname}${release}${shared_ext}$versuffix ${libname}${release}${shared_ext}$major $libname${shared_ext}'
-  soname_spec='${libname}${release}${shared_ext}$major'
+  library_names_spec='${libname}${release}${shared_ext}'
+  soname_spec='${libname}${release}${shared_ext}'
   finish_cmds='PATH="\$PATH:/sbin" ldconfig -n $libdir'
   shlibpath_var=LD_LIBRARY_PATH
   shlibpath_overrides_runpath=no
