Index: clutter-1.0.8/clutter/cogl/common/cogl-material.c
===================================================================
--- clutter-1.0.8.orig/clutter/cogl/common/cogl-material.c	2010-11-18 20:10:25.852000062 +0900
+++ clutter-1.0.8/clutter/cogl/common/cogl-material.c	2010-11-18 20:11:09.028000068 +0900
@@ -1553,8 +1553,8 @@
 
 #ifndef HAVE_COGL_GLES /* GLES 1 only has glBlendFunc */
       if (material->blend_src_factor_rgb != material->blend_src_factor_alpha
-          || (material->blend_src_factor_rgb !=
-              material->blend_src_factor_alpha))
+          || (material->blend_dst_factor_rgb !=
+              material->blend_dst_factor_alpha))
         {
           if (have_blend_equation_seperate &&
               material->blend_equation_rgb != material->blend_equation_alpha)
