Index: libbind-6.0/dst/dst_internal.h
===================================================================
--- libbind-6.0.orig/dst/dst_internal.h
+++ libbind-6.0/dst/dst_internal.h
@@ -19,7 +19,7 @@
  */
 #include <limits.h>
 #include <sys/param.h>
-#if (!defined(BSD)) || (BSD < 199306)
+#if !defined(__ANDROID__) && ((!defined(BSD)) || (BSD < 199306))
 # include <sys/bitypes.h>
 #else
 # include <sys/types.h>
Index: libbind-6.0/include/arpa/inet.h
===================================================================
--- libbind-6.0.orig/include/arpa/inet.h
+++ libbind-6.0/include/arpa/inet.h
@@ -64,7 +64,7 @@
 /* External definitions for functions in inet(3) */
 
 #include <sys/param.h>
-#if (!defined(BSD)) || (BSD < 199306)
+#if !defined(__ANDROID__) && ((!defined(BSD)) || (BSD < 199306))
 # include <sys/bitypes.h>
 #else
 # include <sys/types.h>
Index: libbind-6.0/include/arpa/nameser.h
===================================================================
--- libbind-6.0.orig/include/arpa/nameser.h
+++ libbind-6.0/include/arpa/nameser.h
@@ -60,7 +60,7 @@
 #define BIND_4_COMPAT
 
 #include <sys/param.h>
-#if (!defined(BSD)) || (BSD < 199306)
+#if !defined(__ANDROID__) && ((!defined(BSD)) || (BSD < 199306))
 # include <sys/bitypes.h>
 #else
 # include <sys/types.h>
Index: libbind-6.0/include/netdb.h
===================================================================
--- libbind-6.0.orig/include/netdb.h
+++ libbind-6.0/include/netdb.h
@@ -94,7 +94,7 @@
 
 #include <sys/param.h>
 #include <sys/types.h>
-#if (!defined(BSD)) || (BSD < 199306)
+#if !defined(__ANDROID__) && ((!defined(BSD)) || (BSD < 199306))
 # include <sys/bitypes.h>
 #endif
 #include <sys/cdefs.h>
Index: libbind-6.0/include/resolv.h
===================================================================
--- libbind-6.0.orig/include/resolv.h
+++ libbind-6.0/include/resolv.h
@@ -57,7 +57,7 @@
 #define	_RESOLV_H_
 
 #include <sys/param.h>
-#if (!defined(BSD)) || (BSD < 199306)
+#if !defined(__ANDROID__) && ((!defined(BSD)) || (BSD < 199306))
 # include <sys/bitypes.h>
 #else
 # include <sys/types.h>
Index: libbind-6.0/port_after.h.in
===================================================================
--- libbind-6.0.orig/port_after.h.in
+++ libbind-6.0/port_after.h.in
@@ -25,7 +25,7 @@
 #include <sys/socket.h>
 #include <sys/param.h>
 #include <sys/time.h>
-#if (!defined(BSD)) || (BSD < 199306)
+#if !defined(__ANDROID__) && ((!defined(BSD)) || (BSD < 199306))
 #include <sys/bitypes.h>
 #endif
 #ifdef HAVE_INTTYPES_H
