From 8d282563df7cd235f5d76c06eb15dcd7d354c468 Mon Sep 17 00:00:00 2001
From: "Craig R. Hughes" <craig.git@rungie.com>
Date: Mon, 19 Sep 2011 09:30:00 -0700
Subject: [PATCH] Fix calculation of duration for ClutterState default
 transitions

---
 clutter/clutter-state.c |    9 +++++++--
 1 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/clutter/clutter-state.c b/clutter/clutter-state.c
index 0f38668..9a80abe 100644
--- a/clutter/clutter-state.c
+++ b/clutter/clutter-state.c
@@ -1905,8 +1905,13 @@ clutter_state_get_duration (ClutterState *state,
   if (target_state != NULL)
     {
       if (source_state_name)
-        ret = GPOINTER_TO_INT (g_hash_table_lookup (target_state->durations,
-                                                    source_state_name));
+        {
+          ret = GPOINTER_TO_INT (g_hash_table_lookup (target_state->durations,
+                                                      source_state_name));
+          if(!ret)
+            ret = GPOINTER_TO_INT (g_hash_table_lookup (target_state->durations,
+                                                        NULL));
+        }
       else
         ret = GPOINTER_TO_INT (g_hash_table_lookup (target_state->durations,
                                                     NULL));
-- 
1.7.6

