Index: cogl-1.8.2/cogl/cogl-renderer.c
===================================================================
--- cogl-1.8.2.orig/cogl/cogl-renderer.c
+++ cogl-1.8.2/cogl/cogl-renderer.c
@@ -161,7 +161,7 @@ cogl_renderer_check_onscreen_template (C
   if (!winsys->renderer_connect (renderer, error))
     return FALSE;
 
-  display = cogl_display_new (renderer, onscreen_template);
+/*  display = cogl_display_new (renderer, onscreen_template);
   if (!cogl_display_setup (display, error))
     {
       cogl_object_unref (display);
@@ -169,6 +169,7 @@ cogl_renderer_check_onscreen_template (C
     }
 
   cogl_object_unref (display);
+  */
 
   return TRUE;
 }
