Index: cogl-1.8.2/cogl/winsys/cogl-winsys-egl.c
===================================================================
--- cogl-1.8.2.orig/cogl/winsys/cogl-winsys-egl.c	2012-01-30 13:37:53.271786055 -0800
+++ cogl-1.8.2/cogl/winsys/cogl-winsys-egl.c	2012-01-30 13:47:43.847786014 -0800
@@ -72,6 +72,12 @@
 #define COGL_ONSCREEN_X11_EVENT_MASK StructureNotifyMask
 #endif
 
+#ifdef COGL_HAS_EGL_PLATFORM_POWERVR_NULL_SUPPORT
+/* Borrowing this for Trickplay */
+extern NativeWindowType tp_egl_get_native_window();
+extern NativeDisplayType tp_egl_get_native_display();
+#endif
+
 typedef enum _CoglEGLWinsysFeature
 {
   COGL_EGL_WINSYS_FEATURE_SWAP_REGION                   =1L<<0,
@@ -461,7 +467,7 @@
     wl_display_iterate (egl_renderer->wayland_display,
                         egl_renderer->wayland_event_mask);
 #else
-  egl_renderer->edpy = eglGetDisplay (EGL_DEFAULT_DISPLAY);
+  egl_renderer->edpy = eglGetDisplay (tp_egl_get_native_display());
 
   status = eglInitialize (egl_renderer->edpy,
 			  &egl_renderer->egl_version_major,
@@ -884,7 +890,7 @@
   egl_display->egl_surface =
     eglCreateWindowSurface (edpy,
                             config,
-                            (NativeWindowType) NULL,
+                            tp_egl_get_native_window(),
                             NULL);
   if (egl_display->egl_surface == EGL_NO_SURFACE)
     {
