Index: cogl-1.8.2/cogl/winsys/cogl-winsys-egl.c
===================================================================
--- cogl-1.8.2.orig/cogl/winsys/cogl-winsys-egl.c
+++ cogl-1.8.2/cogl/winsys/cogl-winsys-egl.c
@@ -217,12 +217,10 @@ _cogl_winsys_renderer_get_proc_address (
 {
   void *ptr;
 
-  ptr = eglGetProcAddress (name);
-
-  /* eglGetProcAddress doesn't support fetching core API so we need to
-     get that separately with GModule */
-  if (ptr == NULL)
-    g_module_symbol (renderer->libgl_module, name, &ptr);
+  if(!g_module_symbol(renderer->libgl_module, name, &ptr))
+  {
+    ptr = eglGetProcAddress (name);
+  }
 
   return ptr;
 }
