Index: fontconfig-2.8.0/doc/Makefile.am
===================================================================
--- fontconfig-2.8.0.orig/doc/Makefile.am
+++ fontconfig-2.8.0/doc/Makefile.am
@@ -21,9 +21,11 @@
 # TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
 # PERFORMANCE OF THIS SOFTWARE.
 
-CC = @CC_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
-LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
+COMPILE = $(CC_FOR_BUILD) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
+		      $(CPPFLAGS_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD)
+LINK = $(CC_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD) $(AM_LDFLAGS) $(LDFLAGS_FOR_BUILD) -o $@
+LIBS = $(LIBS_FOR_BUILD)
 
 DOC_SRC = $(srcdir)
 DOC_MODULE = fontconfig
Index: fontconfig-2.8.0/fc-arch/Makefile.am
===================================================================
--- fontconfig-2.8.0.orig/fc-arch/Makefile.am
+++ fontconfig-2.8.0/fc-arch/Makefile.am
@@ -22,9 +22,11 @@
 # PERFORMANCE OF THIS SOFTWARE.
 #
 
-CC = @CC_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
-LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
+COMPILE = $(CC_FOR_BUILD) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
+	$(CPPFLAGS_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD)
+LINK = $(CC_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD) $(AM_LDFLAGS) $(LDFLAGS_FOR_BUILD) -o $@
+LIBS = $(LIBS_FOR_BUILD)
 
 INCLUDES=-I${top_srcdir}/src -I${top_srcdir} $(WARN_CFLAGS)
 
Index: fontconfig-2.8.0/fc-case/Makefile.am
===================================================================
--- fontconfig-2.8.0.orig/fc-case/Makefile.am
+++ fontconfig-2.8.0/fc-case/Makefile.am
@@ -22,9 +22,11 @@
 # PERFORMANCE OF THIS SOFTWARE.
 #
 
-CC = @CC_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
-LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
+COMPILE = $(CC_FOR_BUILD) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
+		      $(CPPFLAGS_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD)
+LINK = $(CC_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD) $(AM_LDFLAGS) $(LDFLAGS_FOR_BUILD) -o $@
+LIBS = $(LIBS_FOR_BUILD)
 
 INCLUDES=-I${top_srcdir}/src -I${top_srcdir} $(WARN_CFLAGS)
 
Index: fontconfig-2.8.0/fc-glyphname/Makefile.am
===================================================================
--- fontconfig-2.8.0.orig/fc-glyphname/Makefile.am
+++ fontconfig-2.8.0/fc-glyphname/Makefile.am
@@ -22,9 +22,11 @@
 # PERFORMANCE OF THIS SOFTWARE.
 #
 
-CC = @CC_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
-LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
+COMPILE = $(CC_FOR_BUILD) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
+		      $(CPPFLAGS_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD)
+LINK = $(CC_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD) $(AM_LDFLAGS) $(LDFLAGS_FOR_BUILD) -o $@
+LIBS = $(LIBS_FOR_BUILD)
 
 INCLUDES=-I${top_srcdir}/src -I${top_srcdir} $(WARN_CFLAGS)
 
Index: fontconfig-2.8.0/fc-lang/Makefile.am
===================================================================
--- fontconfig-2.8.0.orig/fc-lang/Makefile.am
+++ fontconfig-2.8.0/fc-lang/Makefile.am
@@ -22,9 +22,11 @@
 # PERFORMANCE OF THIS SOFTWARE.
 #
 
-CC = @CC_FOR_BUILD@
 EXEEXT = @EXEEXT_FOR_BUILD@
-LINK = $(CCLD) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
+COMPILE = $(CC_FOR_BUILD) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
+		      $(CPPFLAGS_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD)
+LINK = $(CC_FOR_BUILD) $(AM_CFLAGS) $(CFLAGS_FOR_BUILD) $(AM_LDFLAGS) $(LDFLAGS_FOR_BUILD) -o $@
+LIBS = $(LIBS_FOR_BUILD)
 
 INCLUDES=-I${top_srcdir}/src -I${top_srcdir} $(WARN_CFLAGS)
 
