Index: glib-2.30.2/Makefile.am
===================================================================
--- glib-2.30.2.orig/Makefile.am
+++ glib-2.30.2/Makefile.am
@@ -6,7 +6,7 @@ include $(top_srcdir)/Makefile.decl
 
 ACLOCAL_AMFLAGS = -I m4macros ${ACLOCAL_FLAGS}
 
-SUBDIRS = . m4macros glib gmodule gthread gobject gio tests po docs
+SUBDIRS = . m4macros glib gmodule gthread gobject gio po
 DIST_SUBDIRS = $(SUBDIRS) build
 
 bin_SCRIPTS = glib-gettextize
Index: glib-2.30.2/gio/Makefile.am
===================================================================
--- glib-2.30.2.orig/gio/Makefile.am
+++ glib-2.30.2/gio/Makefile.am
@@ -276,8 +276,6 @@ giowin32include_HEADERS = \
 
 endif
 
-SUBDIRS += tests
-
 libgio_2_0_la_SOURCES =		\
 	gappinfo.c 		\
 	gasynchelper.c 		\
Index: glib-2.30.2/glib/Makefile.am
===================================================================
--- glib-2.30.2.orig/glib/Makefile.am
+++ glib-2.30.2/glib/Makefile.am
@@ -42,9 +42,9 @@ gregex_c =
 gregex_h =
 endif
 
-SUBDIRS = libcharset $(PRINTF_SUBDIR) $(MAYBE_PCRE) update-pcre . tests
+SUBDIRS = libcharset $(PRINTF_SUBDIR) $(MAYBE_PCRE) update-pcre .
 
-DIST_SUBDIRS = libcharset gnulib pcre update-pcre tests
+DIST_SUBDIRS = libcharset gnulib pcre update-pcre
 
 AM_CPPFLAGS = 				\
 	$(glib_INCLUDES) 		\
Index: glib-2.30.2/gobject/Makefile.am
===================================================================
--- glib-2.30.2.orig/gobject/Makefile.am
+++ glib-2.30.2/gobject/Makefile.am
@@ -4,7 +4,7 @@
 ## Process this file with automake to produce Makefile.in
 include $(top_srcdir)/Makefile.decl
 
-SUBDIRS = . tests
+SUBDIRS = .
 
 BUILT_SOURCES=
 CLEANFILES=
Index: glib-2.30.2/gthread/Makefile.am
===================================================================
--- glib-2.30.2.orig/gthread/Makefile.am
+++ glib-2.30.2/gthread/Makefile.am
@@ -1,8 +1,7 @@
 ## Process this file with automake to produce Makefile.in
 include $(top_srcdir)/Makefile.decl
 
-SUBDIRS = . tests
-DIST_SUBDIRS = tests
+SUBDIRS = .
 
 AM_CPPFLAGS = 				\
 	$(glib_INCLUDES) 		\
